client/src/relatedProducts/YourOutfitView/YourOutfitCard.jsx

```diff
@@ -27,13 +27,25 @@ const YourOutfitCard = (props) => {
   return (
     <div className='yourOutfitCard'>
         <div className='relatedProductsCardTop'>
-          <h2 className='productName' >{props.outfitProps.name}</h2>
+          <h3 className='productName' >{props.outfitProps.name}</h3>
           <button className='yourOutfitActionButton' onClick={(e) => {props.handleRemoveFromOutfit(props.outfitProps, e)}}>X</button>
         </div>
         <div className='relatedProductsCardBottom'>
-          <h3 className='productCategory'>{props.outfitProps.category}</h3>
-          <h3 className='originalProductPrice'>{props.outfitProps.originalPrice}</h3>
-          <h3 className='saleproductPrice'>{props.outfitProps.salePrice}</h3>
+          <h4 className='productCategory'>{props.outfitProps.category}</h4>
+          {(function() {
```

Comment:
this code works fine - but if you found sometime instead of returning twice - you could attach a boolean to state.  When this state changes, it can change your class in whatever html you need to show, in stead of hide.

this would clean up the if/else and the two return statements.  Not needed just a suggestion!